save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/BuilderComponent.kt

```diff
@@ -42,10 +39,23 @@ external interface BuilderComponentProps : Props {
     "LongMethod",
     "TYPE_ALIAS"
 )
-private fun builderComponent() = FC<BuilderComponentProps> { props ->
-    val (selectedLanguage, setSelectedLanguage) = useState(AceModes.KOTLIN)
+val cpgDemoComponent = FC<BuilderComponentProps> { props ->
```

Comment:
This was planned to be a common component for various future demos, not just CPG. Where the code window will be on the right and the custom component on the left.

Reply:
and now it is not due to a hook with request